pkg/config/config.go

```diff
@@ -186,7 +191,7 @@ func (operatorConfig *NSXOperatorConfig) GetTokenProvider() auth.TokenProvider {
 }
 
 func (operatorConfig *NSXOperatorConfig) createTokenProvider() error {
-	log.V(1).Info("try to load VC host CA")
+	log.Debug("try to load VC host CA")
```

Comment:
please still keep log.V(1) or log.Info, we don't need two kinds logger format in our code.